bundle.js

```diff
@@ -19,38 +20,69 @@ function deleteDirectory(directory) {
     }
 }
 
-function replaceFileContents(string, fileDir) {
-    const pattern = /%%([^%]+)%%|\/\/\s*include\s*"([^"]+)"/g;
+/**
+ * Process file content by resolving includes and translation placeholders.
+ *
+ * @param {string} fileContent original file content
+ * @param {string} fileDir path of the directory of the file
+ * @param {string} locale language code which should be used to resolve translation placeholders
+ * @returns {string}
+ */
+function replaceFileContents(fileContent, fileDir, locale) {
+    const pattern = /%%([^%]+)%%|__(.+?)__/g;
 
-    return string.replace(pattern, (_, filename1, filename2) => {
-        const filename = filename1 || filename2;
-        const filenamePath = path.join(fileDir, filename);
-        try {
-            const data = fs.readFileSync(filenamePath, "utf8");
-            const replacedData = replaceFileContents(data, path.dirname(filenamePath));
-            return replacedData;
-        } catch (error) {
-            console.error(`Error reading file "${filenamePath}":`, error);
-            return "";
+    return fileContent.replace(pattern, (_, filename, translationKey) => {
+        if (filename != undefined) {
+            const filenamePath = path.join(fileDir, filename);
+            try {
+                const data = fs.readFileSync(filenamePath, "utf8");
+                const replacedData = replaceFileContents(data, path.dirname(filenamePath), locale);
+                return replacedData;
+            } catch (error) {
+                console.error(`Error reading file "${filenamePath}":`, error);
+                return "";
+            }
+        } else if (translationKey != undefined) {
+            return i18n.translateWithLocale(locale, translationKey);
         }
     });
 }
 
+/**
+ * Copy inputFile to outputFile, possibly modifying it in the process.
+ *
+ * @param {string} inputFile path
+ * @param {string} outputFile path
+ * @param {function(string, boolean, string):boolean} filter takes path, whether it is a directory and data (or null if directory), returns true if the file should be left out
+ */
 function processFile(inputFile, outputFile, filter) {
-    const fileDir = path.dirname(inputFile);
-    if (inputFile.includes(".mp3")) {
```

Comment:
I have inverted the logic here.

Previously this function special cased mp3 as the only binary file on which the processing should not be done. But right now the tags are used only in HTML files and I don't see why would we want to run it on any other file:  CSS and JS is handled differently and there are no other text file types.